content/articles/loading-image-with-glide-in-android/index.md

```diff
@@ -0,0 +1,605 @@
+alt: Loading Image with Glide in Android
+---
+
+In this tutorial, we are going to focus on how to use the Glide library to load images either from the internet(URL) or Drawable image file into an `ImageView`.
+
+### Prerequisites
+
+To understand this tutorial, the reader should:
+
+- Have Android Studio installed
+
+- Have an understanding of Kotlin programming language and XML.
+
+- Have a basic understanding of image loading in android applications.
+
+- Be familiar with `ViewBinding`. 
```

Comment:
Consider removing the extra blank lines. Do the same in the entire article where applicable.